### PR TITLE
Fix win32 build

### LINKED ISSUE #1: Add win32 support
Have the NIF be compiled and usable on win32.

### PR DESCRIPTION
**Pull request by tolbrino**
Fixes #1

### CODE REVIEW

**Review by hanssv — APPROVED** [review]

File: c_src/emcl_nif.c [hunk 1]
```diff
@@ -297,14 +303,16 @@ ERL_NIF_TERM enif_mcl_bn_fr_sqr(ErlNifEnv *env, int argc, ERL_NIF_TERM const arg
...
 static
 ERL_NIF_TERM enif_mcl_bn_fr_sqrt(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
-  mclBnFr x, z;
+  mclBnFr x;
+  mclBnFr z;
 
-  if(!get_fr(env, argv[0], &x))
+  if(!get_fr(env, argv[0], &x)) {
     return enif_make_badarg(env);
+}
```
Comment (hanssv) [inline c_src/emcl_nif.c:311]: Indentation?

File: c_src/emcl_nif.c [hunk 2]
```diff
@@ -297,14 +303,16 @@ ERL_NIF_TERM enif_mcl_bn_fr_sqr(ErlNifEnv *env, int argc, ERL_NIF_TERM const arg
 
 static
 ERL_NIF_TERM enif_mcl_bn_fr_sqrt(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
-  mclBnFr x, z;
+  mclBnFr x;
+  mclBnFr z;
```
Comment (hanssv) [inline c_src/emcl_nif.c:307]: Hmm... If you are payed per-line, but otherwise?? 🤔

File: test/emcl_tests.erl [hunk 3]
```diff
@@ -171,18 +171,18 @@ gt_test() ->
...
-  {ok, G2} = emcl:bnG2_hash_and_map_to(<<"abc">>),
-  GT       = emcl:bnGt_from_str(<<"1 2 3 4 5 6 7 8 9 10 11 12">>),
-
-  {ok, _} = emcl:bnG1_mul_vec(lists:duplicate(5, G1), Frs),
-  {ok, _} = emcl:bnG2_mul_vec(lists:duplicate(5, G2), Frs),
-  {ok, _} = emcl:bnGt_pow_vec(lists:duplicate(5, GT), Frs),
-
-  ok.
-
+%mul_vec_test() ->
```
Comment (hanssv) [inline test/emcl_tests.erl]: I think these tests are fine now?

**Review by hanssv — CHANGES_REQUESTED** [review]
I think all tests should be run?

File: test/emcl_tests.erl (hunk 3 above)
Comment (tolbrino) [reply test/emcl_tests.erl]: Fixed.

**Review by hanssv — APPROVED** [review]

**Pull request merged**